Index: xbase-clients/xedit/xedit.man
===================================================================
--- xbase-clients.orig/xedit/xedit.man	2006-02-18 14:56:14.000000000 -0500
+++ xbase-clients/xedit/xedit.man	2006-02-26 20:37:32.000000000 -0500
@@ -291,7 +291,7 @@
 .B autoReplace (\fPClass \fBReplace)
 This resource is useful to automatically correct common misspelling errors, but
 can also be used to create simple macros. The format is
-\fI{non-blanks}{blanks}[{string}]\fP. Fields are separeted by newlines.
+\fI{non-blanks}{blanks}[{string}]\fP. Fields are separated by newlines.
 Example of use:
 \"
 .br
@@ -308,7 +308,7 @@
 when you type that words.
 .TP 8
 .B ispell.dictionaries (\fPClass \fBispell.Dictionary)
-Specifies a list of dictionary names, separeted by spaces, available to the
+Specifies a list of dictionary names, separated by spaces, available to the
 ispell program. The default value is \fI"american americamed+ english"\fP.
 .TP 8
 .B ispell.dictionary (\fPClass \fBispell.Dictionary)
